monitors/didyoumean.js

```diff
@@ -18,8 +18,10 @@ exports.run = async (client, msg) => {
 
   const distances = [];
 
+  const highPerm = calcHighPerm(client, msg);
+
   for (const cmd in client.commands) {
```

Comment:
remove this and usableCommands extended property. Does exactly what you want.

Reply:
another reason why this should be changed: 
usablecommands runs all of the inhibitors, while the previous code doesn't, so you could accidentially expose hidden commands or something similar